app/i18n/en/sub.php

```diff
@@ -221,6 +221,14 @@
 		'title' => 'Title',
 		'title_add' => 'Add an RSS feed',
 		'ttl' => 'Do not automatically refresh more often than',
+		'unicityPolicy' => array(
+			'_' => 'Articles unicity',
```

Comment:
Maybe something like:
```suggestion
			'_' => 'Article uniqueness criteria',
```
Or:
```suggestion
			'_' => 'Article duplicate matching criteria',
```

Which is a long way of saying perhaps `unicity` isn't the most common word.

Reply:
Interesting ChatGPT's feedback 🤖
![image](https://github.com/user-attachments/assets/2961bba6-dded-4121-8ce3-7d5f7dbb7a56)

Indeed not the most common word but not so rare either, is it ? And probably "guessable". I do not really mind, but *unicity* is slightly more precise here.

It is important to be understandable indeed. On the other hand, this option should not be too tempting to fiddle with :-P

Let's see what our testers say, if any

Reply:
Either way I would make it article (singular) unicity. ^_^